src/compute/src/render/join/delta_join.rs

```diff
@@ -324,11 +324,11 @@ where
     CF: Fn(&G::Timestamp, &G::Timestamp) -> bool + 'static,
 {
     match trace {
-        SpecializedArrangement::Bytes9Row(key_types, inner) => build_halfjoin(
+        SpecializedArrangement::RowUnit(inner) => build_halfjoin(
```

Comment:
Uncovered

Reply:
Same as above; please reopen this conversation if not true.